.travis.yml

```diff
@@ -55,17 +54,18 @@ matrix:
     - os: osx
       # TODO: xcode8 triggers undefined symbol 'clock_gettime' in boost::interprocess
       osx_image: xcode7.3
-      compiler: clang
+      compiler: "mason-osx-release"
+      # we use the xcode provides clang and don't install our own
       env: ENABLE_MASON=ON BUILD_TYPE='Release' CUCUMBER_TIMEOUT=60000 CCOMPILER='clang' CXXCOMPILER='clang++' ENABLE_ASSERTIONS=ON
 
     # Release Builds
     - os: linux
-      compiler: "mason-release"
+      compiler: "mason-linux-release"
       addons:
         apt:
           sources: ['ubuntu-toolchain-r-test']
           packages: ['libstdc++-5-dev']
-      env: BUILD_TYPE='Release' ENABLE_MASON=ON
+      env: CLANG_VERSION='3.8.1' BUILD_TYPE='Release' ENABLE_MASON=ON
```

Comment:
Just keeps the current version we use on master. Upgrading should be a separate PR. This fixes a problem where we didn't install clang because this wasn't set.